### ISSUE TITLE
migrate to travis-ci.com

### ISSUE THREAD

**Issue by slivingston** [tulip-control/tulip-control#210]
As presented [on their website](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/), open source projects will eventually be moved to travis-ci.com. The task is to complete this migration as described at https://docs.travis-ci.com/user/migrate/open-source-repository-migration

checklist for doing this in tulip-control related repositories:
- [x] Tulip (here)
- [x] dd (https://github.com/tulip-control/dd)
- [x] Polytope (https://github.com/tulip-control/polytope)
- [x] gr1c
- [x] check that not too many repositories from tulip-control organization were added to travis-ci.com (or if so, is it of any consequence?)

**Comment by slivingston** [comment]
actually, this is still in "beta" as described at https://docs.travis-ci.com/user/migrate/open-source-repository-migration
so I will close this issue and re-open when the time is right.

**Comment by slivingston** [comment]
The future is upon us. I just migrated this repository to travis-ci.com and pushed 2 small documentation-only commits that successfully triggered CI jobs: https://travis-ci.com/github/tulip-control/tulip-control/builds/227465703

**Comment by johnyf** [comment]
Thank you for migrating the CI of the repository to travis-ci.com. The transition of the CI for the repositories `gr1c`, `tulip-control`, `polytope`, `omega`, and `dd` to travis-ci.com has now been completed. The repositories `cudd` and `data` also appear on `travis-ci.com`, I do not know why (when I migrated the CI entries of other repositories, I remember that I only selected the specific repositories I was about to migrate). In any case, regarding the last item in the checklist above (https://github.com/tulip-control/tulip-control/issues/210#issue-429080777), I think that there is no issue with having these additional repositories present in travis-ci.com.
